src/api/ProductModel.js

```diff
@@ -105,7 +105,7 @@ function getEssentialMeasurements(itemTypeArr) {
     switch (itemTypeArr[0]) {
         case 1:
             arr.push("chest", "front_height");
-            if (itemTypeArr[3] >= 6) {
+            if ( (itemTypeArr[3] >= 6) && (itemTypeArr[2] == 1) ) {
```

Comment:
It's recommended to use operator `===` when comparing values, to make sure the values are of the same type. For example: `"1" == 1` is true, but `"1" === 1` is false. In this case it doesn't matter, but it's good practice to always use `===`.